### ISSUE TITLE
Add a metadata description table

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/ioos-atn-data#25]
We should take the [template](https://github.com/ioos/ioos-atn-data/blob/main/templates/atn_trajectory_template.cdl) and build a table which:
1. Provides a description for each attribute.
2. Identifies which standard/convention/recommendation we got that attribute from.
3. Identify the Type of attribute (global vs variable)
4. Identify the role in the template (required/recommended)

**Comment by mmckinzie** [comment]
@MathewBiddle I can start working on this...I have to do the same thing for our AniBOS template

**Comment by MathewBiddle** [comment]
Thanks, @mmckinzie. I was going to start it this AM. Obviously that didn't happen. Let me know how I can help.

**Comment by mmckinzie** [comment]
@MathewBiddle I have been working on the description table. Available [[here](https://docs.google.com/spreadsheets/d/169DIXCD-YeTGm9MEmxFhwaVGf-M02YKT/edit?usp=sharing&ouid=113417505623695767850&rtpof=true&sd=true)]. So far only includes global attributes, I still need to add in the variables.

**Comment by MathewBiddle** [comment]
Lets publish this as a documentation site similar to https://ioos.github.io/ioos-metadata/ioos-metadata-profile-v1-2.html. I can start work on this soon.

**Comment by MathewBiddle** [comment]
Okay I have the website updated to reflect how I think the documentation should be organized. 

https://ioos.github.io/ioos-atn-data/index.html

There is a high level **ATN Specification** page which documents and links to all the various ATN specifications we might develop. Right now we only have the Satellite Telemetry Specification, so it looks a little bare bones. But, the idea is this will be the main page we point folks to who might be looking for animal tracking data templates 

Underneath the **ATN Specification** page there will be subsequent pages for the different specifications, like ARGOS, DTAG, trajectory profile, and whatever else there might be. There will also be pages for the different versions of each specification. These will be appropriately documented in the sections of the **ATN Specification** page, as well as the sidebar menu. 

For example, when we develop v2 of the satellite trajectory template, that will be a new page linked on the sidebar and appropriately referenced on the **ATN Specification** page.

As far as documenting what we have at the moment, I have started a page for the [**ATN Satellite Telemetry Specification Version 0.1**](https://ioos.github.io/ioos-atn-data/atn-sat-telem-specification-v0-1.html) (I made it version 0.1 because the documentation is not appropriate at this time and I don't want folks to get confused, we will update to v1.0 once it's ready to be released). I lifted the organization of this page heavily from the IOOS Metadata Profile page, so I'm curious as to the most efficient way to present our template.

I like how the IOOS Metadata Profile splits the information into appropriate sections, I'm just not sure what those sections should be in our case. Another option would be to make one gigantic table that lists out all the attributes, describes them, and identifies their source. Any opinions are welcome on that front.
